api/types/container/host_config.go

```diff
@@ -243,8 +243,10 @@ type Resources struct {
 	BlkioDeviceWriteBps  []*blkiodev.ThrottleDevice
 	BlkioDeviceReadIOps  []*blkiodev.ThrottleDevice
 	BlkioDeviceWriteIOps []*blkiodev.ThrottleDevice
-	CPUPeriod            int64           `json:"CpuPeriod"` // CPU CFS (Completely Fair Scheduler) period
-	CPUQuota             int64           `json:"CpuQuota"`  // CPU CFS (Completely Fair Scheduler) quota
+	CPUPeriod            int64           `json:"CpuPeriod"`    // CPU CFS (Completely Fair Scheduler) period
+	CPUQuota             int64           `json:"CpuQuota"`     // CPU CFS (Completely Fair Scheduler) quota
+	CPURtPeriod          int64           `json:"CpuRtPeriod"`  // CPU real-time period
```

Comment:
I followed the same capitalization as runc had for real time, and followed your pattern for CPU: https://github.com/opencontainers/runc/blob/7d8f322fdd85c19a8b92f920ce96f91bb5a370b7/libcontainer/configs/cgroup_unix.go#L75

Happy to do `CPURTPeriod` of `CPURealTimePeriod` if it's prefered